### ISSUE TITLE
Add pending status to the values of the feature status

### ISSUE THREAD

**Issue by andieri** [wswebcreation/protractor-multiple-cucumber-html-reporter-plugin#18]
**Environment (please complete the following information):**
 - protractor-multiple-cucumber-html-reporter: 1.7.1
 - Node.js version: 6.10.2
 - NPM version: 3.10.10
 - Platform name and version: Windows 10
 - protractor version: 5.3.0
 - Cucucmber version: 3.2.1
 - protractor-cucumber-framework version: 4.2.0

**Config of protractor + protractor-multiple-cucumber-html-reporter**
           options: {
                    automaticallyGenerateReport: true,
                    customData: {
                        title: 'Execution information',
                        data: []
                    },
                    displayDuration: true,
                    durationInMS: true,
                    pageTitle: 'Functional E2E Report',
                    reportName: 'Functional E2E Report',
                    reportPath: './reports',
                    removeExistingJsonReportFile: true,
                    jsonOutputPath: './reports/resources'
                }

**Describe the bug**
I have pending steps in the test code. When I run the whole test set, the scenarios which contains pending steps are mark as pending. But the Feature file status is passed. 

**To Reproduce**
1. Execute a tests, which have pending steps
2. Open the automaticly generated test report
3. Find the pending scenario and see the feature status.

**Expected behavior**
In this case I think this not a valid information about the feature status. The status of the feature can be pending too.

**Additional context**
![pending scenarios passes](https://user-images.githubusercontent.com/15669716/43949746-b907299c-9c8e-11e8-985f-c867c2c19a97.PNG)

**Comment by wswebcreation** [comment]
Hi @andieri

Tnx for filling the issue. I'm currently not able to check the issue due to vacation.
Keep you posted.

**Comment by wswebcreation** [comment]
Hi @andieri 

This has been fixed in the new version of `multiple-cucumber-html-reporter`. The examples can be found on https://wswebcreation.github.io/multiple-cucumber-html-reporter/browsers/index.html.

I've created a new release, which can be found [here](https://github.com/wswebcreation/protractor-multiple-cucumber-html-reporter-plugin/releases/tag/v1.8.0). Just update your dependencies.

Hope it fits your needs. I'm going to close it for now but it is still open for questions
